EnglishFilter/sections/foreign.txt

```diff
@@ -6158,6 +6158,8 @@ roweroweporady.pl,tulodz.pl#@#.ad_container
 !+ PLATFORM(ios, ext_android_cb)
 @@||lib.wtg-ads.com/prebid/prebid_*.js$domain=tvn.pl
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/105421
+||niebezpiecznik.pl/wp-content/uploads/*/600x500.jpg
```

Comment:
I don't see it. Have you had it on your end?

Reply:
Banner is changed I guess. https://imgur.com/a/nuut43i

Reply:
Banner is changed I guess. https://imgur.com/a/nuut43i

Reply:
Now it's a self-promo, added rules directly.